src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -743,7 +743,7 @@ The job search panel allows to select multi-value filters for the following job
 
 - _Workflow Name(s)_: Jobs can be filtered by workflow name. Selecting/Typing one or more workflow names is provided by a built-in `auto-complete` feature that helps you search for workflows or buckets from the link:../user/ProActiveUserGuide.html#_catalog_portal[ProActive Catalog.]
 - _Project Name(s)_: You can also filter by one or more project names. You just have to specify the project names for the jobs you would like to analyze.
-- _Job State(s)_: You can specify the state of jobs you are looking for. The possible job states are: `Pending`, `Running`, `Stalled`, `Paused`, `In_Error`, `Finished`, `Canceled`, `Failed`, and `Killed`. For more information about job states, check the documentation link:../user/ProActiveUserGuide.html#_states[here.] Multiple values are accepted as well.
+- _Job Status(es)_: You can specify the state of jobs you are looking for. The possible job statuses are: `Pending`, `Running`, `Stalled`, `Paused`, `In_Error`, `Finished`, `Canceled`, `Failed`, and `Killed`. For more information about job statuses, check the documentation link:../user/ProActiveUserGuide.html#_statuses[here.] Multiple values are accepted as well.
```

Comment:
Status can be used for plural, I think it will be more readable if you use status instead of status(es)